### PR TITLE
Ft githooks

### PR DESCRIPTION
**Pull request by MartinaLof**

### CODE REVIEW

**Comment by nonguyen1** [comment]
So for clang-format, it is a dependency which must be install separately. This installation can be tricky to implement through a setup.sh script. I suggest we use ESLint because its just a node module which can be install easily with npm install. They offer the same support, syntax checking, and auto fixes when we choose the same style guides.

As for using git hooks it might be more beneficial to just use Husky, as it does the pre-commit execution for us rather than having to write a git hooks. Again, helps us avoid the setup script.

This guide might be useful
https://www.orangejellyfish.com/blog/code-consistency-with-eslint-and-husky/

File: package.json [hunk 1]
```diff
@@ -27,13 +40,16 @@
...
     "lit-html": "^1.0.0",
     "mocha": "^6.1.4"
   },
   "dependencies": {
     "@babel/runtime": "^7.4.4",
     "@open-wc/testing": "^0.11.4",
     "@webcomponents/custom-elements": "^1.2.4",
-    "@webcomponents/webcomponentsjs": "^2.2.10"
+    "@webcomponents/webcomponentsjs": "^2.2.10",
+    "clang-format": "^1.2.4",
```
Comment (ShaneNielsen1234) [inline package.json]: Is clang-format still needed if we are using husky? Can this dependency be removed now?

File: setup.sh [hunk 2]
```diff
@@ -1,7 +1,4 @@
 #!/bin/sh
-# setup script that moves commit hook for code formatting to proper directory
-# and sets up unit testing
-cp scripts/pre-commit .git/hooks
```
Comment (ShaneNielsen1234) [inline setup.sh]: Can the file scripts/pre-commit be deleted? Or is it still needed for something?

File: test/hwTest.js [hunk 3]
```diff
@@ -1,36 +0,0 @@
-import {expect, assert} from 'chai';
```
Comment (ShaneNielsen1234) [inline test/hwTest.js]: Why is this showing as deleted? Should this test be deleted from master?

Comment (ShaneNielsen1234) [reply test/hwTest.js]: Same with the bottom two tests.

Comment (nonguyen1) [reply test/hwTest.js]: These can be deleted. `hwTest.js` has been relocated into test/test-core with updated name that matches conventional standard (`test/core-hello/core-hello_test.js`). Other test files were dummy file used by developer to test functionality. Essentially these are duplicated files which can be found in `packages/core-hello/demo`.

File: setup.sh (hunk 2 above)
Comment (nonguyen1) [reply setup.sh]: Yes, these can be deleted now.

File: package.json (hunk 1 above)
Comment (nonguyen1) [reply package.json]: Will still need Husky. This is our git hooks manager instead of using a setup.sh script now.

**Pull request merged**